sensor/kubernetes/enforcer/cronjob/suspend.go

```diff
@@ -51,8 +51,16 @@ func makePatch(deploymentInfo *central.DeploymentEnforcement, apiVersion string)
 
 // Suspend suspends the cron job
 func Suspend(ctx context.Context, client kubernetes.Interface, deploymentInfo *central.DeploymentEnforcement) (err error) {
-	if ok, apiErr := utils.HasAPI(client, batchV1, kubernetesPkg.CronJob); ok && apiErr == nil {
+	ok, apiErr := utils.HasAPI(client, batchV1, kubernetesPkg.CronJob)
+	if apiErr != nil {
+		return retry.MakeRetryable(apiErr)
+	}
```

Comment:
[nit and optional, but...]
Unless there's a good reason, the style which seems dominant is to name `error` as just `err`, not `apiErr` or `somethingErr`.

```suggestion
	ok, err := utils.HasAPI(client, batchV1, kubernetesPkg.CronJob)
	if err != nil {
		return retry.MakeRetryable(err)
	}
```